test/oatpp-openssl/FullAsyncClientTest.cpp

```diff
@@ -162,6 +165,7 @@ class ClientCoroutine_getRootAsync : public oatpp::async::Coroutine<ClientCorout
   }
 
   Action handleError(Error* error) override {
+    SUCCESS_COUNTER = -2;
```

Comment:
This is not strictly part of the verification/validation, but the test would time out if the handshake failed.  Propagate the failure so the test fails quickly.